src/alpha-lab/phone-neon.md

```diff
@@ -10,42 +10,33 @@ tags: [Neon, Cloud]
 <TagLinks />
 <Youtube src="gp5O1uskDME"/>
 
+::: tip
+What catches your attention and how do you visually navigate through the interface/page of a mobile app? This guide shows you the tools you can use to address these questions. 
+:::
 
-Have you ever wondered what your eyes focus on when scrolling through your favourite app? 
-
-In this article, we test whether Neon, our latest eye tracker, can accurately capture and characterise viewing behaviour 
-while gazing at small icons on mobile applications. To achieve this, we used some of Alpha Lab's existing tutorials, including
-how to generate scanpaths, define areas of interest (AOIs) and calculate outcome metrics, and map gaze onto dynamic content.
+## Gaze behaviour during mobile app engagement
+Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we tested the effectiveness of our latest wearable eye tracker, Neon, in capturing and analysing users' viewing behaviour when focusing on small icons and features of mobile applications.
```

Comment:
Reading through this again, I suggest we slightly re-frame the intent of the article. Rather than 'testing' the effectiveness of Neon for mobile browsing, we should say that we 'used' neon to investigate mobile browsing. I reason that it wasn't really a proper test. But by demonstrating we used it to examine phone browsing, it shows that it's possible and motivates people to try it, rather than making concrete claims.